sdk_test.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/joho/godotenv"
+)
+
+func TestInitializeClient(t *testing.T) {
+	_ = godotenv.Load(".env")
+	_, err := pubsub.NewClient(context.Background(), ProjectID)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestInitializeTopic(t *testing.T) {
+	client, _ := pubsub.NewClient(context.Background(), ProjectID)
+	defer client.Close()
+
+	topicRef := client.Topic(TopicID)
+	defer topicRef.Stop()
+
+	exists, err := topicRef.Exists(context.Background())
+	if err != nil {
+		t.Error(err)
+	}
+
+	if !exists {
+		_, err = client.CreateTopic(context.Background(), TopicID)
+		if err != nil {
+			t.Error(err)
+		}
+	}
```

Comment:
Shouldn't you be testing your functions? Why are you testing the pubsub library  functions  instead of your own functions?